README.md

```diff
@@ -578,6 +578,17 @@ The [Google Places Details API](https://developers.google.com/places/documentati
 * **Limitations**: ?
 * **Notes**: You can specify which projection you want to use by setting, for example: `Geocoder.configure(:esri => {:outSR => 102100})`.
 
+#### Mapzen/Pelias (`:pelias`)
```

Comment:
Could you split this documentation into two, one for Mapzen and another for Pelias?
And add short sentences to describe each? eg.
"Mapzen is geocoder powered by open data and uses elastic search"
"Pelias is installed locally and uses the Mapzen API"

`:pelias` should be renamed `:pelias_local` to match the other local lookups.